gateway-service-impl/src/main/java/org/hypertrace/gateway/service/logevent/LogEventsService.java

```diff
@@ -0,0 +1,172 @@
+package org.hypertrace.gateway.service.logevent;
+
+import static org.hypertrace.gateway.service.common.util.AttributeMetadataUtil.getTimestampAttributeMetadata;
+
+import com.google.common.collect.ImmutableMap;
+import io.micrometer.core.instrument.Timer;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.hypertrace.core.attribute.service.v1.AttributeMetadata;
+import org.hypertrace.core.query.service.api.ColumnMetadata;
+import org.hypertrace.core.query.service.api.QueryRequest;
+import org.hypertrace.core.query.service.api.ResultSetChunk;
+import org.hypertrace.core.query.service.api.ResultSetMetadata;
+import org.hypertrace.core.query.service.api.Row;
+import org.hypertrace.core.query.service.client.QueryServiceClient;
+import org.hypertrace.core.serviceframework.metrics.PlatformMetricsRegistry;
+import org.hypertrace.gateway.service.common.AttributeMetadataProvider;
+import org.hypertrace.gateway.service.common.RequestContext;
+import org.hypertrace.gateway.service.common.converters.QueryAndGatewayDtoConverter;
+import org.hypertrace.gateway.service.v1.common.OrderByExpression;
+import org.hypertrace.gateway.service.v1.log.events.LogEvent;
+import org.hypertrace.gateway.service.v1.log.events.LogEventsRequest;
+import org.hypertrace.gateway.service.v1.log.events.LogEventsResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LogEventsService {
+
+  private static final Logger LOG = LoggerFactory.getLogger(LogEventsService.class);
+
+  private static final String LOG_EVENT_SCOPE = "LOG_EVENT";
```

Comment:
Should we add the scope here as well? https://github.com/hypertrace/attribute-service/blob/main/attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto#L9

Reply:
It is deprecated, check https://github.com/hypertrace/attribute-service/pull/89#pullrequestreview-636577006